packages/cli/src/prompts/prompt.ts

```diff
@@ -202,8 +171,14 @@ export function methodsList(
     });
 }
 
-export function argLabel(arg: MethodArg): string {
-  return arg.name ? `${arg.name}: ${ABI.getArgTypeLabel(arg)}` : ABI.getArgTypeLabel(arg);
+export function argLabel(arg: MethodArg, index?: number): string {
```

Comment:
Previously `arg.name` was being set to `#${index}` in some other part of the prompts module. Now this function is sometimes used for plain `MethodArg` extracted from the ABI without any preprocessing.

Reply:
I had to undo this because it was being used in a place that I hadn't foreseen and it caused a test to fail.